admin/config/locales/en.yml

```diff
@@ -746,6 +746,21 @@ en:
           save_asset: Save Asset
           tags_label: Tags
           tags_note: 'Comma separated: just, like, this'
+          tags_tooltip_html: |
+            <p>You can specify tags on an asset to mark them for special use cases.</p>
```

Comment:
I'm not sure what markup is best of this context, but regarding content, how about something like this?:

Use the following tags to set an asset as a favicon or Open Graph image for your site:

"favicon" - The default favicon
"favicon-<var>W</var>x<var>H</var>", e.g. "favicon-32x32" - The favicon for the given size
"og-default" - The default Open Graph image

The default favicon is used when a device does not request a specific size or when the requested size is not available.
The default Open Graph image is used when sharing a content page that does not specify an Open Graph image.

Reply:
sounds good to me.